build-tests/api-extractor-test-02/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "module": "commonjs",
     "declaration": true,
     "sourceMap": true,
+    "declarationMap": true,
```

Comment:
Can we add this to the default RSC configs?

Reply:
We absolutely should, but not in this PR